flexget/plugins/sites/newznab.py

```diff
@@ -1,124 +1,561 @@
 from __future__ import unicode_literals, division, absolute_import
 from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
-from past.utils import old_div
-from future.moves.urllib.parse import urlencode, quote
+from time import mktime
 
+from xml.dom import minidom
+from past.builtins import basestring
+from datetime import datetime
+from dateutil.parser import parse as dateutil_parse
+from dateutil.tz import tz
+from future.moves.urllib.parse import quote_plus
+
+import re
 import logging
+import feedparser
 
 from flexget import plugin
 from flexget.entry import Entry
 from flexget.event import event
-import feedparser
+from flexget.config_schema import one_or_more
+from flexget.utils.tools import parse_timedelta
+from flexget.utils.template import render_from_entry, RenderError
+from flexget.utils.search import normalize_unicode
+from flexget.plugins.parsers.parser_common import normalize_name, remove_dirt
```

Comment:
Oh right, it happens within discover plugin, not during task execution. This is ongoing design challenge, it's not right to make quick solutions with one search plugin though.